src/rpc/util.cpp

```diff
@@ -74,29 +75,29 @@ CAmount AmountFromValue(const UniValue& value, int decimals)
     return amount;
 }
 
-uint256 ParseHashV(const UniValue& v, std::string strName)
+uint256 ParseHashV(const UniValue& v, std::string_view name)
```

Comment:
What is the difference of changing `std::string` to `std::string_view` in comparision of changing it to `const std::string& name`? The overhead of a reference to the entire object?

Reply:
Cool, thanks!